src/components/QuizGameQuestions/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./index.css";
+
+const QuestionCard = ({
```

Comment:
Can you add comments on top of this function to explain how it's used? like, where is it called, what does the callback do?

Reply:
PTAL